src/content/learn/preserving-and-resetting-state.md

```diff
@@ -1,44 +1,43 @@
 ---
-title: Preserving and Resetting State
+title: State সংরক্ষণ এবং রিসেট করা
 ---
 
 <Intro>
 
-State is isolated between components. React keeps track of which state belongs to which component based on their place in the UI tree. You can control when to preserve state and when to reset it between re-renders.
+State কম্পোনেন্টের মধ্যে ভিন্ন রাখা হয়। React একটি UI ট্রি এর ভিতরে কোন State কোন কম্পোনেন্টের সঙ্গে সংযোগস্থলে পরিচয় রেখে থাকে। আপনি নিয়ন্ত্রণ করতে পারেন যখন অবস্থা সংরক্ষণ করতে হবে এবং যখন রি-রেন্ডার এর মধ্যে তা পুনরায় সেট করতে হবে। 
```

Comment:
Sentence gulo onek kothin hoyeche. Please take another tab at it and try to write in simple bangla.